dist/src/main/config/broker.yaml.template

```diff
@@ -651,6 +651,7 @@
         #     timer: true
         #     variable: true
         #     variableDocument: true
+        #     form: true
```

Comment:
🔧 I guess this one should be moved up a bit to be in alphabetical order.

Reply:
I'm stupid 😄

Reply:
I still disagree 🙂

Reply:
Thanks for spotting this 👍